packages/utilities/psammead-storybook-helpers/src/withServicesKnob.jsx

```diff
@@ -2,20 +2,41 @@ import React from 'react';
 import { Helmet } from 'react-helmet';
 import { select } from '@storybook/addon-knobs';
 import * as scripts from '@bbc/gel-foundations/scripts';
-import LANGUAGE_VARIANTS from './text-variants';
+import EXAMPLES from './text-variants';
```

Comment:
Reading this I'm constantly reminding myself what `EXAMPLES` is. It needs a better name `TEXT_EXAMPLES` or `TEXT_SNIPPETS` or something